WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -0,0 +1,249 @@
+import UIKit
+import WordPressComAnalytics
+import WordPressShared
+
+/// A base class for the various NUX related related view controllers. 
+/// The base class sets up and configures common functionality, such as the help 
+/// button and badge.
+/// It is assumed that NUX controllers will always be presented modally.
+///
+class NUXAbstractViewController : UIViewController
```

Comment:
This looks like a common _chrome_ for all the NUX controllers. Did you consider making this a container instead of a base class?